### PR TITLE
Directory restructuring of Attachment related components

### PR DESCRIPTION
**Pull request by vishalnarkhede**
# Submit a pull request

## CLA

- [ ] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required).
- [ ] The code changes follow best practices
- [ ] Code changes are tested (add some information if not applicable)

## Description of the pull request

- Moving file type attachment to. its own component - File
- Moving Audio, Card, File to Attachment directory
- Making Audio, Card, File, Media customisable through props.

### CODE REVIEW

File: src/components/Attachment/Attachment.js [hunk 1]
```diff
@@ -147,7 +159,7 @@ class Attachment extends PureComponent {
             key={`key-video-${a.id}`}
           >
             <div className="str-chat__player-wrapper">
-              <ReactPlayer
```
Comment (virdesai) [inline src/components/Attachment/Attachment.js:150]: ReactPlayer being used anywhere else in this component or is that an unused import at the top now?

File: src/components/Attachment/File.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { FileIcon } from 'react-file-utils';
+import prettybytes from 'pretty-bytes';
+import { SafeAnchor } from '../SafeAnchor';
+
+const File = ({ attachment: a }) => {
```
Comment (virdesai) [inline src/components/Attachment/File.js:6]: doesn't seem like this renaming is needed, seems to just obfuscate things a bit

File: src/components/Gallery/index.js [hunk 3]
```diff
@@ -1 +1,2 @@
 export { default as Gallery } from './Gallery';
+export { default as Image } from './Image';
```
Comment (jaapbakker88) [inline src/components/Gallery/index.js:2]: https://github.com/GetStream/stream-chat-react/pull/314

did these also here

File: src/components/Attachment/Attachment.js [hunk 4]
```diff
@@ -1,15 +1,14 @@
 /* eslint-disable */
 import React, { PureComponent } from 'react';
 import ReactPlayer from 'react-player';
```
Comment (jaapbakker88) [inline src/components/Attachment/Attachment.js:3]: Probably want to do `import ReactPlayer as Media`

And then 
```
defaultProps: {
   Media
}
```

**Pull request merged**